test/unit/model/test_model_selection.py

```diff
@@ -79,22 +78,67 @@ def test_split_data_train_validation_test_stratification(self):
         # delta = 4, when looking at the test data, can differ 1 hr (4x15min)
 
         self.assertAlmostEqual(
-            len(peaks_val_train[0][0]),
-            len(peaks) * SPLIT_PARAMS["validation_fraction"],
-            delta=1,
-        )
-        self.assertAlmostEqual(
-            len(peaks_val_train[1][0]),
-            len(peaks) * (1 - SPLIT_PARAMS["validation_fraction"]),
-            delta=1,
-        )
+            len(valid_set),
+            len(data) * SPLIT_PARAMS["validation_fraction"],
+            delta=2 * 96,
+        )  # two days is allowed
 
         self.assertAlmostEqual(
             len(test_set),
             len(data.index) * SPLIT_PARAMS["test_fraction"],
             delta=4,
         )
 
+    def test_split_data_train_validation_test_stratification(self):
```

Comment:
Nice unit test, suggestion: May document in the following way: https://wiki.c2.com/?ArrangeActAssert

Reply:
Good idea